tests/console/umoci.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test umoci installation and basic usage
+#    Cover the following aspects of umoci:
+#      * Inspecting a repository before pulling it
+#      * Copy an image from Docker registry into an OCI image-spec
+#      * Unpack and repack (root/rootless) images
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use strict;
+use base "consoletest";
+use testapi;
+use caasp;
+
+sub run {
+    select_console("root-console");
+
+    record_info 'Test #1', 'Test: Installation';
+    my $package = 'umoci';
+    trup_install($package);
+
+    record_info 'Setup', 'Requirements';
+    assert_script_run('mkdir test_umoci && cd test_umoci');
+    assert_script_run('skopeo copy docker://docker.io/opensuse/tumbleweed oci:opensuse:tumbleweed', 300);
```

Comment:
Please increase timeout to 600 - we had some failures yesterday with 300 - see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4676